### PR TITLE
Release 1.0.0 : update workflows

### PR DESCRIPTION
**Pull request by mmulholla**
This PR was created by:
replacing the contents of :
- .github/
- scripts/
- docs/

adding:
- tests/
- assets/

modifying:
- .github/workflows/schedule.yaml to add the schedule invocation (schedule invocation not included in development).   
- .github/workflows/build.yaml to add use the latest verifier image (development uses main)

### CODE REVIEW

**Comment by mmulholla** [comment]
Completed testing:
partner charts:
  - chart src with report
  - chart tgz and no report
  - chart src and no report

**Review by dperaza4dustbit — APPROVED** [review]
LGTM with a caveat:  as long as we can test a chart flow end to end. Right after merge lets trigger workflow test to make sure it works and distinguishes between chart changes and workflow tests.

**Pull request merged**
